### PR TITLE
fixed invalid JWT signatures generated

### LINKED ISSUE #81: py_vapid: Invalid JWT signatures generated
I'm getting Invalid JWT tokens once in a while when doing webpush and I think it's because they are actually invalid.

This example calculates the signature length:
```
>>> v
<py_vapid.Vapid02 object at 0x7f7ce7900880>
>>> c
Counter()
>>> for _ in range(10000):
...   d = v.sign({'sub': "mailto:horse@example.com", "aud": "https://example.com:8080"})
...   c[len(base64.urlsafe_b64decode(d['Authorization'].split('.')[2].split(',')[0] + '===='))] += 1
>>> c
Counter({64: 9929, 63: 71})
```

My understanding is that it should be 64 bytes long every time.

Edit: Been using my own fix and it solved my issue completely.

### PR DESCRIPTION
**Pull request by JohnDoee**
Trying to fix #81

### CODE REVIEW

File: python/py_vapid/utils.py [hunk 1]
```diff
@@ -26,11 +26,14 @@ def b64urlencode(data):
     return base64.urlsafe_b64encode(data).replace(b'=', b'').decode('utf8')
 
 
-def num_to_bytes(n):
+def num_to_bytes(n, l):
```
Comment (jrconlin) [inline python/py_vapid/utils.py:29]: Looks like flake8 complains about `l` as a variable name. Could you use something like `length` or `pad_to`?

**Comment by jrconlin** [comment]
By the way, I totally get how finicky flake8 can be from version to version, but it's important that the library pass all checks before any patch gets merged. If you don't have time, I can try resubmitting the fix in a different patch when I get the chance.

**Comment by JohnDoee** [comment]
I hope the changes are sufficient, I'd be really happy if this could get merged and released so I don't have to run my own fork.

**Comment by jrconlin** [comment]
Sorry, long weekend and a few work items came up. 

Packaging it up now.

**Pull request merged**